plugin/src/main/java/at/helpch/chatchat/command/WhisperCommand.java

```diff
@@ -0,0 +1,29 @@
+package at.helpch.chatchat.command;
+
+import at.helpch.chatchat.ChatChatPlugin;
+import at.helpch.chatchat.util.FormatUtils;
+import dev.triumphteam.cmd.core.BaseCommand;
+import dev.triumphteam.cmd.core.annotation.Command;
+import dev.triumphteam.cmd.core.annotation.SubCommand;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+@Command("chatchat")
```

Comment:
might be worth making this just its own separate command so people can just do /msg instead of /chatchat msg
if we go for this, the subcommand annotation will need to be replaced with @Default